lib/Basics/errors.dat

```diff
@@ -99,7 +99,7 @@ ERROR_ARANGO_SYNC_TIMEOUT,1111,"sync timeout","Will be raised when the server wa
 ERROR_ARANGO_CONFLICT,1200,"conflict","Will be raised when updating or deleting a document and a conflict has been detected."
 ERROR_ARANGO_DATADIR_INVALID,1201,"invalid database directory","Will be raised when a non-existing database directory was specified when starting the database."
 ERROR_ARANGO_DOCUMENT_NOT_FOUND,1202,"document not found","Will be raised when a document with a given identifier is unknown."
-ERROR_ARANGO_DATA_SOURCE_NOT_FOUND,1203,"collection or view not found","Will be raised when a collection with the given identifier or name is unknown."
+ERROR_ARANGO_DATA_SOURCE_NOT_FOUND,1203,"collection or View not found","Will be raised when a collection with the given identifier or name is unknown."
```

Comment:
This change may break clients or tests. I know it is just a text change, but unfortunately I have seen clients/tests not testing for the return codes but for return strings. This is a very common error code, so I have concerns someone tests on that exact string.

Reply:
Agreed, even some of our own tests do this.

BTW. The long description doesn't mention Views yet, will fix that but revert the change to the error text.